js/performance.ts

```diff
@@ -24,6 +20,6 @@ export class Performance {
     assert(msg.Any.NowRes === baseRes.innerType());
     const res = new msg.NowRes();
     assert(baseRes.inner(res) != null);
-    return res.time().toFloat64() - this.timeOrigin;
+    return res.seconds().toFloat64() * 1000 + res.subsecNanos() / 1000;
```

Comment:
I realize I suggested this, but I think there' an error here, it should be:

res.seconds().toFloat64() * 1e3 + res.subsecNanos() / 1e6

❗️

Reply:
Right for the nano subsec